pkgs/development/tools/vagrant/0004-Support-system-installed-plugins.patch

```diff
@@ -0,0 +1,96 @@
+From: Antonio Terceiro <terceiro@debian.org>
```

Comment:
is this an upstream patch?

Reply:
No, it's from Debian: https://sources.debian.org/src/vagrant/2.1.5+dfsg-1/debian/patches/0004-Support-system-installed-plugins.patch/

I made a couple of modifications to make it work with Nix.

Reply:
Done